### DISCUSSION TITLE
Enable auto review request ONLY if all tests have passed

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#9158] [Pull Requests]
We have multiple tests running for each PR.
We do not want to have a colleague review the changes unless all the tests have passed.
Currently, what I do is open the PR without any reviewers, and only if all the tests pass, I go back to the PR page and request reviews.
By "all test", I mean all tests that are required to allow merging the PR.

**It would be great to implement the same concept of the `Enable auto-merge`👇🏼**  
![image](https://user-images.githubusercontent.com/23265867/146889390-8065df35-b559-4ec7-b648-8556e5124e1f.png)

**So when you open a PR, you can set it to send the review requests only if all tests have passed.** 👇🏼 
![image](https://user-images.githubusercontent.com/23265867/146891703-17836944-fe9b-4dde-9e32-be3ddc4be51b.png)

**Comment by TarunavBA** [comment]
@Avivhdr I think this is not a god request since many PR's are necessary but they don't always pass all the test so I think this request is not possible to implement.

**Reply by Avivhdr** [reply]
Hi @TarunavBA 🙂
The request is to make it optional, not mandatory (same as `Enable auto-merge`).
So if you have a PR that you know/suspect will not pass all tests, you could still request the review immediately.
